app/models/comment.rb

```diff
@@ -264,6 +265,11 @@ def create_first_reaction
     Comments::CreateFirstReactionJob.perform_later(id)
   end
 
+  def after_destroy_actions
+    Users::BustCacheJob.perform_now(user_id)
+    user&.touch(:last_comment_at)
```

Comment:
Is there a scenario where the comment's user is nil?

Reply:
Yes, but in tests.
I came across this error  : "Comment validations should validate that ... "   while running the following spec `rspec ./spec/models/comment_spec.rb:33` (is_expected.to validate_uniqueness_of(:body_markdown) etc …

Turns out the subject(comment) has been created without a user.

If I replace
`subject { described_class.new(commentable: article) }`
with:
`subject { described_class.new(user: user, commentable: article) }`

Another error arose then, so I had to update that same spec :
`let(:article) { Article.new }`
with
`let(:article) { Article.new(user: user2) }`
Otherwise the method parent_user in comment would return nil

With these, there is no need to add the safe operator & in both cases and tests run ok.

I do not think there is some impact on the rest of the comment specs.

Do you want me to update the comment spec this way ?

Reply:
Oki, I have changed in this way.